source/guide_jekyll.rst

```diff
@@ -0,0 +1,210 @@
+.. highlight:: console
+
+.. author:: Pascal Nowak <pascal@nowak.app>
+
+.. sidebar:: Logo
+
+
+  .. image:: _static/images/jekyll.png
+      :align: center
+
+
+#########
+Jekyll
+#########
+
+Jekyll_ is an easy to use static site generator that builds fast, blog-aware, responsive websites. 
+
+The website is created with markdown and Jekyll generates the html files and deploys it to the webserver. 
+It is possible to connect Jekyll to git, so that the website can be created on a local machine and then pushed to git where it gets deployed automatically via git hooks.
+
+Jekyll is written in Ruby and licensed under the MIT license.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`Ruby <lang-ruby>`
+  * git_
+  * :manual:`Domains <web-domains>`
+
+License
+=======
+
+All relevant legal information can be found here 
+
+  * https://github.com/jekyll/jekyll/blob/master/LICENSE
+
+Prerequisites
+=============
+We're using :manual:`Ruby <lang-ruby>` in the version 2.5.3:
+
+::
+
+ [isabell@stardust ~]$ ruby -v
+ ruby 2.5.3p105 (2018-10-18 revision 65156) [x86_64-linux]
+ [isabell@stardust ~]$
+
+Your domain needs to be setup:
+
+.. include:: includes/web-domain-list.rst
+
+You need to install a gem called Jekyll:
+::
+
+ [isabell@stardust ~]$ gem install bundler jekyll
+ Fetching bundler-2.0.1.gem
+ [...]
+ Successfully installed jekyll-3.8.5
+ 26 gems installed
+ [isabell@stardust ~]$
+
+Installation
+============
+Jekyll is a ruby gem that creates dummy files and directories for us that contain your website. We can modify these files to make your website look like we want it to.
+
+Just let Jekyll create a new folder containing your website by typing:
+::
+
+ [isabell@stardust ~]$ jekyll new website
```

Comment:
Just of out curiosity: this already seems to install the needed gems. They are even present in `~/website/vendor/bundle/gems/`, but `bundle exec` can't find the `jekyll` command. Do you happen to know why?

Reply:
Hey, I am currently on vacation and have no possibility to check that out on my uberspace. They blocked ssh ports on guest wifi. I will have a look at it, when I am back.

Reply:
I tried to reproduce your question and figured out, that on a local machine the command `jekyll new website` is enough to execute `bundle exec jekyll serve` and everything works fine.
But on uberspace I have to type `bundle install --path vendor/bundle` to install missing gems after creating the new website. 
But I dont know why the gems seem to be missing. 

I try to figure out why and if I find a reason, I will comment again.